src/bin/cargo/commands/yank.rs

```diff
@@ -30,14 +30,11 @@ pub fn exec(gctx: &mut GlobalContext, args: &ArgMatches) -> CliResult {
         args.get_one::<String>("crate").map(String::as_str),
         args.get_one::<String>("version").map(String::as_str),
     )?;
-    if version.is_none() {
```

Comment:
Moving this into the function would ideally be a refactor commit on its own

Reply:
Moved the refactoring into its own commit.